show-transaction.md

```diff
@@ -7,6 +7,8 @@ toc: false
 
 The `SHOW TRANSACTION` [statement](sql-statements.html) shows you the current transaction's [priority](transactions.html#transaction-priorities) or [isolation level](transactions.html#isolation-levels).
 
+{{site.data.alerts.callout_info}}<code>SHOW TRANSACTION</code> can be run outside of transactions but simply returns <code>NORMAL</code> priority and <code>SERIALIZABLE</code> isolation level.{{site.data.alerts.end}}
```

Comment:
It returns whatever the current defaults are, right? Above you changed default_transaction_isolation and it was reflected in `show transaction isolation level`.